res/TensorFlowLiteRecipes/CumSum_000/test.recipe

```diff
@@ -0,0 +1,45 @@
+operand {
```

Comment:
plz add `test.reverse` to both recipes and check tflchef reverse is working OK.

Reply:
Ah, I didn't check what .reverse file is for and just ignored them because all of them were empty. 
Now I found it works like as a flag to determine whether applying tflite-reverse test.
I'll add .reverse file to enable tflite-reverse test either.
FYI, I've checked it by manual `(e.g. tflchef-reverse test.tflite test.recipe)`, and it was working.